### PR TITLE
Added metalsmith undeploy command in documentation

### PR DESCRIPTION
**Pull request by skanthed**

### CODE REVIEW

File: docs/source/usage/cli.rst [hunk 1]
```diff
@@ -170,11 +170,15 @@ Image
...
-|                | **Actions**                                                                                                         |
-+----------------+---------------------------------------------------------------------------------------------------------------------+
-| Provision Node | ``metalsmith deploy --resource-class baremetal --image <image> --network <network> --ssh-public-key <path-to-key>`` |
-+----------------+---------------------------------------------------------------------------------------------------------------------+
++-------------------+---------------------------------------------------------------------------------------------------------------------+
+|                   | **Actions**                                                                                                         |
++-------------------+---------------------------------------------------------------------------------------------------------------------+
+| Provision Node    | ``metalsmith deploy --resource-class baremetal --image <image> --network <network> --candidate <node-uuid> --ssh-public-key <path-to-key>`` |
```
Comment (tzumainn) [inline docs/source/usage/cli.rst:176]: I'm pretty sure this spacing breaks the generated table; you can test this on your local machine by  creating a virtualenv and installing the requirements, going into the doc/ directory, and running ``make html``. That will generate html files which you can open up locally in your browser.

File: docs/source/usage/cli.rst [hunk 2]
```diff
@@ -170,11 +170,15 @@ Image
...
++-------------------+---------------------------------------------------------------------------------------------------------------------+
+| Remove Node       | ``metalsmith undeploy <node-uuid>``                                                                                 |
++----------- -------+---------------------------------------------------------------------------------------------------------------------+
+| Remove Deployment | ``metalsmith undeploy <deployment-uuid>``                                                                           |
```
Comment (DanNiESh) [inline docs/source/usage/cli.rst:180]: Where can we get the deployment-uuid?

Comment (skanthed) [reply docs/source/usage/cli.rst:180]: @DanNiESh deployment-uuid and allocation-uuid are same.

File: docs/source/usage/cli.rst (hunk 1 above)
Comment (skanthed) [reply docs/source/usage/cli.rst:176]: Sure I will check that.

File: docs/source/usage/cli.rst (hunk 2 above)
Comment (DanNiESh) [reply docs/source/usage/cli.rst:180]: I only see metalsmith undeploy <NODE UUID> in "metalsmith --help". I've never used undeploy <deployment-uuid> before. Have you tried it out?

Comment (skanthed) [reply docs/source/usage/cli.rst:180]: Yes. I have tried and tested it multiple times. You can also try it on the demo server.

Comment (tzumainn) [inline docs/source/usage/cli.rst:180]: I've always just used `metalsmith undeploy <node id>`, to be honest - does that work?

Comment (skanthed) [reply docs/source/usage/cli.rst:180]: Yes. it works for sure.

**Comment by skanthed** [comment]
@tzumainn As we discussed, Just to avoid confusion added <node id> to undeploy command.  Which encompasses both node uuid and node name.

File: docs/source/usage/cli.rst [hunk 3]
```diff
@@ -170,11 +170,13 @@ Image
...
-|                | **Actions**                                                                                                         |
-+----------------+---------------------------------------------------------------------------------------------------------------------+
-| Provision Node | ``metalsmith deploy --resource-class baremetal --image <image> --network <network> --ssh-public-key <path-to-key>`` |
-+----------------+---------------------------------------------------------------------------------------------------------------------+
++----------------+---------------------------------------------------------------------------------------------------------------------------------------------+
+|                | **Actions**                                                                                                                                 |
++----------------+---------------------------------------------------------------------------------------------------------------------------------------------+
+| Provision Node | ``metalsmith deploy --resource-class baremetal --image <image> --candidate <node-uuid> --network <network> --ssh-public-key <path-to-key>`` |
```
Comment (tzumainn) [inline docs/source/usage/cli.rst:176]: I'm pretty sure that the node name works here as well, so <node id> could be used here

Comment (skanthed) [reply docs/source/usage/cli.rst:176]: Here also I will mention <node id> instead

File: docs/source/usage/cli.rst [hunk 4]
```diff
@@ -170,11 +170,13 @@ Image
...
-+----------------+---------------------------------------------------------------------------------------------------------------------+
-| Provision Node | ``metalsmith deploy --resource-class baremetal --image <image> --network <network> --ssh-public-key <path-to-key>`` |
-+----------------+---------------------------------------------------------------------------------------------------------------------+
++----------------+--------------------------------------------------------------------------------------------------------------------+
+|                | **Actions**                                                                                                        |
++----------------+--------------------------------------------------------------------------------------------------------------------+
+| Provision Node | ``metalsmith deploy --resource-class baremetal --image <image> --network <network> --ssh-public-key <path-to-key>``|
++----------------+---------------------------------------- ---------------------------------------------------------------------------+
```
Comment (DanNiESh) [inline docs/source/usage/cli.rst:177]: I think the table is not rendered correctly.

**Review by tzumainn — APPROVED** [review]

File: docs/source/usage/cli.rst (hunk 4 above)
Comment (tzumainn) [reply docs/source/usage/cli.rst:177]: Oh whoops, Danni is right - you need a space before the last '|'. Can you just make sure this is fixed in the other metalsmith candidate PR?

Comment (skanthed) [reply docs/source/usage/cli.rst:177]: Sure.

**Pull request merged**